app/controllers/concerns/client_sortable.rb

```diff
@@ -1,6 +1,6 @@
 module ClientSortable
   def filtered_and_sorted_clients(default_order: nil)
-    @default_order = default_order || { "id" => "desc" }
+    @default_order = default_order || { "first_unanswered_incoming_interaction_at" => "asc" }
```

Comment:
cool, i actually like this because it was kind of bothering me how the default order was literally just what the db does by default, and thus seemed a little unneccessary. But now that we are using it for something other than the default, I'm glad its there so it was straightforward to customize!